### ISSUE TITLE
House of Words color notations

### ISSUE THREAD

**Issue by xinemata** [Kcaden20/critical-code-cookbook#7]
The original [House of Words submission](https://docs.google.com/document/d/1OzYDAfCm5pxIQfaI_WRg3Ngb8Cz1WsvIvV6S6KucOgs/edit?usp=sharing) uses orange, green, and blue texts to explain the template literal syntax. I wonder what is the best way to recreate this on the [webpage](https://criticalcode.recipes/contributions/house-of-words)?
<img width="637" alt="Screen Shot 2022-06-21 at 2 27 42 PM" src="https://user-images.githubusercontent.com/9159424/174891543-f3f10556-1122-4766-9e2c-383db87bf4be.png">

**Comment by Kcaden20** [comment]
How often does this happen? Is it only in this contribution?

**Comment by Kcaden20** [comment]
@xinemata I'd say that if it's only a few instances, this might be where in-line styling makes sense. Markdown can't communicate color information. 

You could also add a class with that color's name and I could add it into the `style.css` file at a later time if that's easier.

**Comment by xinemata** [comment]
Just tried the inline method! Tbh it's a bit gnarly but perhaps that's alright? Or if you think there's a better way to do it please let me know. 😬

`` <code  style ="padding: 0 0 auto 0;">var sentence =</code><code style ="color: #ff7e0d; padding: 0 0;">\`</code><code  style ="padding: 0 0;">My</code><code style="color: #38761d; padding: 0 0;">$</code><code style="color: #4a86e8; padding: 0 0;">{</code><code  style ="padding: 0 0;">pet</code><code style="color: #4a86e8; padding: 0 0;">}</code><code  style ="padding: 0 0;">eats twice a day.</code><code  style ="color: #ff7e0d; padding: 0 0;">\`</code><code  style ="padding: 0 0;">;</code> ``

<img width="441" alt="Screen Shot 2022-06-26 at 2 38 57 PM" src="https://user-images.githubusercontent.com/9159424/175833024-1ed4a625-0001-4cb6-af1d-19b6770711ec.png">

I am curious though how the screenreader would experience this. FWIW if this starts to feel like too much I could also reach out to let the contributor know that we need to remove the color references from the tutorial.

**Comment by Kcaden20** [comment]
This is pretty gnarly and would be a mess for a screen reader to translate IMO. We'd probably have to hide it all with `aria-hidden` and then add a wrapper around it with the `aria-label`,

The only thing I can think of is creating color tags for each of the highlights (orange, green, blue) and then applying them manually followed by wrapping everything in what we want the screen reader to read out. 

Any other solutions seem to be importing a JS library:
- https://ace.c9.io/#nav=embedding
- https://highlightjs.org/

Also the orange doesn't read very well on the light theme, would there be another color we could use?

**Comment by xinemata** [comment]
Thanks for doing the research. I'd prefer to avoid `aria=hidden` if possible. IMO it's ok to get rid of color codings. I'll just reach out to Megan to let her know that this conflicts with accessibility needs.

**Comment by Kcaden20** [comment]
Agreed. Let me know her thoughtd
*Kevin Cadena*
*Designer, Web Developer & Teacher*
***@***.**** ***@***.***>
*kevincadena.com* <http://kevincadena.com>

**Comment by xinemata** [comment]
Morgan confirmed that it's ok without syntax highlighting. Closing the issue.
